From c44813c1abb45467fa9e07bbde01d1fb09cae383 Mon Sep 17 00:00:00 2001
From: Alexandre Rostovtsev <tetromino@gentoo.org>
Date: Wed, 23 Nov 2011 17:41:12 -0500
Subject: [PATCH] Use $RSVG_API_VERSION in the .gir filename
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

$RSVG_API_VERSION == "2.0"
$RSVG_API_VERSION_U == "2_0"

We want to use 2.0 when referring to the .gir filename, and 2_0 when
referring to the automake targets derived from the filename. Otherwise,
parallel make fails.

Thanks to Rafał Mużyło <galtgendo@o2.pl> for pointing this out in
https://bugs.gentoo.org/show_bug.cgi?id=391215#c10

https://bugzilla.gnome.org/show_bug.cgi?id=664684
---
 Makefile.am |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 511df68..5a8dda3 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -235,7 +235,7 @@ INTROSPECTION_GIRS = Rsvg-@RSVG_API_VERSION@.gir
 INTROSPECTION_SCANNER_ARGS = --add-include-path=$(srcdir)
 INTROSPECTION_COMPILER_ARGS = --includedir=$(srcdir)
 
-Rsvg-@RSVG_API_VERSION_U@.gir: librsvg-@RSVG_API_MAJOR_VERSION@.la
+Rsvg-@RSVG_API_VERSION@.gir: librsvg-@RSVG_API_MAJOR_VERSION@.la
 
 Rsvg_@RSVG_API_VERSION_U@_gir_NAMESPACE = Rsvg
 Rsvg_@RSVG_API_VERSION_U@_gir_EXPORT_PACKAGES = librsvg-$(RSVG_API_VERSION)
-- 
1.7.8.rc3

